src/managers/default/index.js

```diff
@@ -32,7 +32,8 @@ class DefaultViewManager {
 			axis: this.settings.axis,
 			layout: this.layout,
 			width: 0,
-			height: 0
+			height: 0,
+			globalOptions: extend({}, options.settings, {disableSrcdoc: options.settings.disableSrcdoc || false})
```

Comment:
Oh yes, my mistake - this makes sense since you are passing that in from the rendition.